packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -608,20 +608,17 @@ Please provide a valid TCP port (an integer between 0 and 65535, inclusive).
   }) async {
     if (_chromiumLauncher != null) {
       final Chromium chrome = await _chromiumLauncher!.connectedInstance;
-      ChromeTab? chromeTab;
-      try {
-        chromeTab = await getChromeTabGuarded(
-          chrome.chromeConnection,
-          (ChromeTab chromeTab) {
-            return !chromeTab.url.startsWith('chrome-extension');
-          },
-          retryFor: const Duration(seconds: 5),
-        );
-      } on HttpException catch (error, stackTrace) {
-        // We were unable to unable to communicate with Chrome.
-        _logger.printError(error.toString(), stackTrace: stackTrace);
-        chromeTab = null;
-      }
+      final ChromeTab? chromeTab = await getChromeTabGuarded(
+        chrome.chromeConnection,
+        (ChromeTab chromeTab) {
+          return !chromeTab.url.startsWith('chrome-extension');
+        },
+        retryFor: const Duration(seconds: 5),
+        onError: (Object error, StackTrace stackTrace) {
```

Comment:
does this now mean we're swallowing all errors?

Reply:
oh, it looks like this only gets delegated IOException and StateError?

Reply:
oh yeah. Anything that's not `IOException` or `StateError` will fall through. Maybe it should just catch everything.

Reply:
Addressed in [refactor: add clarifying docs](https://github.com/flutter/flutter/pull/154366/commits/9fd3e846b7d2cb8761637839cbca769d38e3f0f3). I decided to just rename the parameter to not imply that any error would be caught.